cli/nodeinfo.ts

```diff
@@ -36,6 +37,12 @@ export const command = new Command()
   )
   .option("-u, --user-agent <string>", "The custom User-Agent header value.")
   .action(async (options, host: string) => {
+    if (Deno.args[0] === "node") {
+      console.warn(
```

Comment:
It's trivial, but for better visibility:


```suggestion
      console.warn("Warning: `fedify node` will be deprecated in Fedify 2.0.0. Use `fedify nodeinfo` instead.")
```